Code/src/main/java/nl/utwente/group10/ui/handlers/ConnectionCreationManager.java

```diff
@@ -0,0 +1,91 @@
+package nl.utwente.group10.ui.handlers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+import java.util.Optional;
+
+import javafx.geometry.Point2D;
+import javafx.scene.Parent;
+import nl.utwente.group10.ui.CustomUIPane;
+import nl.utwente.group10.ui.components.Connection;
+import nl.utwente.group10.ui.components.ConnectionAnchor;
+import nl.utwente.group10.ui.components.ConnectionLine;
+import nl.utwente.group10.ui.components.InputAnchor;
+import nl.utwente.group10.ui.components.OutputAnchor;
+
+public class ConnectionCreationManager {
+
+	CustomUIPane pane;
+
+	/**
+	 * Touch points have an ID associated with each specific touch point, this
+	 * is the ID associated with the Mouse.
+	 */
+	public static final Integer MOUSE_ID = 0;
+	/**
+	 * Maps an (Touch or Mouse) ID to a line, used to keep track of what touch
+	 * point is dragging what line.
+	 */
+	private Map<Integer, Connection> connections;
```

Comment:
Dat doet iets anders dan ik doe he.
Ik doe in feite het volgende:
                map.put(1, null);
		map.put(1, null);
		map.put(1, null);